website/common/locales/en/tasks.json

```diff
@@ -1,4 +1,4 @@
-{
+﻿{
```

Comment:
Looks like an extraneous change. Not a big deal, but it would be a cleaner PR if this file were left out.